### PR TITLE
Chapter 8: Basic login

### PR DESCRIPTION
**Pull request by tungbai121**

### CODE REVIEW

**Comment by tungbai121** [comment]
Chapter 8: Basic login

**Comment by tungbai121** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      redirect_to user
+    else
+      flash[:danger] = "Invalid email/password combination!"
```
Comment (manhbnt-0960) [inline app/controllers/sessions_controller.rb:10]: i18n e

**Review by manhbnt-0960 — COMMENTED** [review]
fix rồi merge

**Pull request merged**